src/framework/mocha/MochaRuntime.h

```diff
@@ -347,6 +347,7 @@
 - (void)shutdown;
 
 - (JSValueRef)JSValueForObject:(id)object;
+- (JSValueRef)JSValueForObject:(id)object shouldCreateBox:(BOOL)shouldCreateBox;
```

Comment:
I think it's worth adding some docs here noting what the `shouldCreateBox` argument does and why it would be YES or NO